internal/scorecard/alpha/labels_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"testing"
+
+	"github.com/ghodss/yaml"
```

Comment:
```suggestion
	"gopkg.in/yaml.v3"
```

Same here. IHMO we need to be very careful with the deps/modules added in the project and we should avoid use any module which is not really maintained and adopted by the community.